libbeat/docs/step-configure-kibana-endpoint.asciidoc

```diff
@@ -0,0 +1,20 @@
+. If you plan to use the sample Kibana dashboards provided with {beatname_uc},
+configure the Kibana endpoint:
++
+[source,yaml]
+----------------------------------------------------------------------
+setup.kibana:
+  host: "localhost:5601"
+----------------------------------------------------------------------
++
+--
+Where `host` is the hostname and port of the machine where Kibana is running,
+for example, `localhost:5601`.
+
+NOTE: If you specify a path after the port number, you need to include
+the scheme and port: `http://localhost:5601/path`.
+
+Also see the security-related options in <<setup-kibana-endpoint>> if you are
+connecting to a secured instance of Kibana.
+
```

Comment:
There are a few more options for configuring Kibana, the same options as for configuring Elasticsearch output.

```
# Optional protocol and basic auth crendentials.
  #protocol: "https"
  #username: "elastic"
  #password: "changeme"

  # Optional HTTP Path
  #path: ""

  # Use SSL settings for HTTPS. Default is true.
  #ssl.enabled: true

  # Configure SSL verification mode. If `none` is configured, all server hosts
  # and certificates will be accepted. In this mode, SSL based connections are
  # susceptible to man-in-the-middle attacks. Use only for testing. Default is
  # `full`.
  #ssl.verification_mode: full

  # List of supported/valid TLS versions. By default all TLS versions 1.0 up to
  # 1.2 are enabled.
  #ssl.supported_protocols: [TLSv1.0, TLSv1.1, TLSv1.2]

  # SSL configuration. By default is off.
  # List of root certificates for HTTPS server verifications
  #ssl.certificate_authorities: ["/etc/pki/root/ca.pem"]

  # Certificate for SSL client authentication
  #ssl.certificate: "/etc/pki/client/cert.pem"

  # Client Certificate Key
  #ssl.key: "/etc/pki/client/cert.key"

  # Optional passphrase for decrypting the Certificate Key.
  #ssl.key_passphrase: ''

  # Configure cipher suites to be used for SSL connections
  #ssl.cipher_suites: []

  # Configure curve types for ECDHE based cipher suites
  #ssl.curve_types: []
```